### PR TITLE
Student signup flow

### PR DESCRIPTION
**Pull request by jkeat**
Links
-----
- Server repo PR: https://github.com/UPchieve/server/pull/413

Description
-----------
- Update student signup flow

Developer self-review checklist
-------------------------------
- [ ] Potentially confusing code has been explained with comments
- [ ] No warnings or errors have been introduced; all known error cases have been handled
- [ ] Any appropriate documentation (within the code, README.md, docs, etc) has been updated
- [ ] All edge cases have been addressed

### CODE REVIEW

**Comment by jkeat** [comment]
@treykris Going to use this PR to address this: https://upchieve.monday.com/boards/438837796/pulses/651953228
> volunteers dont show up on review page unless background info completed

which means there will only be one place for approval to happen, the admin route

File: src/views/SignupView/StudentForm.vue [hunk 1]
```diff
@@ -488,36 +518,8 @@ export default {
...
-      })
-        .then(() => {
-          this.step = "step-1-waitlist-success";
-          this.$router.push("/sign-up/student/waitlist-success");
-        })
-        .catch(err => {
-          this.msg = err.message;
-        });
+    ineligibleContinue() {
+      this.$router.push("https://upchieve.org/parents");
```
Comment (treykris) [inline src/views/SignupView/StudentForm.vue:522]: Nice work!! It looks great

This part here took me to:
parenturl

Comment (jkeat) [reply src/views/SignupView/StudentForm.vue]: Oo yeah good catch

**Comment by jkeat** [comment]
@ me: fix e2e tests

**Pull request merged**